### ISSUE TITLE
Standings Traversal broken

### ISSUE THREAD

**Issue by prcutler** [prcutler/nflpool#152]
Now that we have multiple seasons of NFLPool, the Standings traversal is broken when updating to a new season past the first one.

The Standings Service needs to be updated.  It currently looks like:

```
def get_seasons():
    session = DbSessionFactory.create_session()
    season_row = session.query(SeasonInfo).filter(SeasonInfo.id == '1').first()
    current_season = season_row.current_season

    return current_season
```

This only returns the current season - the function needs to be re-written to get the matching ID from the season the user clicks on.

**Comment by prcutler** [comment]
This is fixed for the Season standings in commit b11047f9c573af8816dd300904af2a26b6df2b17

However, need to test the player details sub-pages - I have a feeling they're broken based on the way the season is passed to those sub-pages

**Comment by prcutler** [comment]
Looking at the code again, this is fixed and I don't think the player detail sub-pages will be affected as it inherits from the season that is passed.
